api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -25,7 +25,7 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#extension: envoy.transport_sockets.tls]
 // The TLS contexts below provide the transport socket configuration for upstream/downstream TLS.
 
-// [#next-free-field: 8]
+// [#next-free-field: 9]
 message UpstreamTlsContext {
```

Comment:
Is this something we might also need in `DownstreamTlsContext` for the server side?